extension-types.md

```diff
@@ -38,18 +38,34 @@ metadata).
 ## Extension metadata
 
 When GeoArrow-encoded Arrays have the `ARROW:extension:metadata` metadata
-field set, it must be seriaized as a UTF-8 encoded JSON object. The following
-keys in the JSON metadata object are supported:
-
-- `crs`: A JSON object describing the coordinate reference system (CRS)
-  using [PROJJSON](https://proj.org/specifications/projjson.html).
-  This key can also be omitted if the producer does not have any
-  information about the CRS. Note that regardless of the axis
-  order specified by the CRS, axis order will be interpreted
-  according to the wording in the
-  [GeoPackage WKB binary encoding](https://www.geopackage.org/spec130/index.html#gpb_format):
-  axis order is always (longitude, latitude) and (easting, northing)
-  regardless of the the axis order encoded in the CRS specification.
+field set, it must be seriaized as a UTF-8 encoded JSON object. The extension
+metadata specification is intentionally aligned with the
+[GeoParquet column metadata specifification](https://github.com/opengeospatial/geoparquet/blob/main/format-specs/geoparquet.md#metadata).
+The following keys in the JSON metadata object are supported:
+
+- `crs`: One of:
+
+    - A JSON object describing the coordinate reference system (CRS)
+      using [PROJJSON](https://proj.org/specifications/projjson.html).
+      This key can also be omitted if the producer does not have any
+      information about the CRS. Note that regardless of the axis
+      order specified by the CRS, axis order will be interpreted
+      according to the wording in the
+      [GeoPackage WKB binary encoding](https://www.geopackage.org/spec130/index.html#gpb_format):
+      axis order is always (longitude, latitude) and (easting, northing)
+      regardless of the the axis order encoded in the CRS specification.
```

Comment:
I would move this "Note that regardless .." down outside the "One of" sublist, since this is relevant for all different ways you can specify the CRS.